x-pack/plugins/observability/public/application/index.tsx

```diff
@@ -31,27 +32,25 @@ function getTitleFromBreadCrumbs(breadcrumbs: Breadcrumbs) {
 
 function App() {
   return (
-    <>
-      <Switch>
-        {Object.keys(routes).map((key) => {
-          const path = key as keyof typeof routes;
-          const route = routes[path];
-          const Wrapper = () => {
-            const { core } = usePluginContext();
+    <Switch>
+      {Object.keys(routes).map((key) => {
+        const path = key as keyof typeof routes;
+        const route = routes[path];
+        const Wrapper = () => {
+          const { core } = usePluginContext();
 
-            const breadcrumb = [observabilityLabelBreadcrumb, ...route.breadcrumb];
-            useEffect(() => {
-              core.chrome.setBreadcrumbs(breadcrumb);
-              core.chrome.docTitle.change(getTitleFromBreadCrumbs(breadcrumb));
-            }, [core, breadcrumb]);
+          const breadcrumb = [observabilityLabelBreadcrumb, ...route.breadcrumb];
+          useEffect(() => {
+            core.chrome.setBreadcrumbs(breadcrumb);
+            core.chrome.docTitle.change(getTitleFromBreadCrumbs(breadcrumb));
+          }, [core, breadcrumb]);
 
-            const { query, path: pathParams } = useRouteParams(route.params);
-            return route.handler({ query, path: pathParams });
-          };
-          return <Route key={path} path={path} exact={true} component={Wrapper} />;
-        })}
-      </Switch>
-    </>
+          const { query, path: pathParams } = useRouteParams(path);
+          return route.handler({ query, path: pathParams });
+        };
+        return <Route key={path} path={path} exact={true} component={Wrapper} />;
```

Comment:
Maybe we should extract `Wrapper` here - inlining might mean `Route` re-renders more often than necessary.